.github/workflows/deploy-dev.yml

```
@@ -1,171 +1,171 @@
+
+      # Log in the local Docker client
+      - name: Login to Amazon ECR
+        id: login-ecr-build
+        uses: aws-actions/amazon-ecr-login@v2
+
+      # Set output variable tag with the current checked out ref
+      - name: Set Image Url
+        id: image-url
+        env:
+          TARGET_IMAGE_URL: ${{ steps.login-ecr-build.outputs.registry }}/${{ vars.ECR_REPOSITORY }}:${{ github.ref_name || github.ref || github.sha }}
+        run: echo "image_url=${TARGET_IMAGE_URL}" >> $GITHUB_OUTPUT
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:41: Double quote to prevent globbing and word splitting

(shellcheck)